solutions/swb-reference/integration-tests/support/clientSession.ts

```diff
@@ -18,6 +18,7 @@ export default class ClientSession {
   private _axiosInstance: AxiosInstance;
   private _setup: Setup;
   private _userId?: string;
+  public accessToken?: string;
```

Comment:
Should accessToken be accessed this easily? More secure to store it as a private variable and return the TTL when needed instead of passing the access token around

Reply:
Since this is for integ tests, I don't think it should be an issue to provide access, has potential to help with debugging to know who did what and with what groups available.